examples/examples/rpc_middleware_modify_request.rs

```diff
@@ -57,6 +59,14 @@ where
 
 		self.0.call(req)
 	}
+
+	fn batch(&self, batch: Batch<'a>) -> Self::Future {
+		self.0.batch(batch)
```

Comment:
A valid impl might be good here to show that you can't ignore methods in batches either?

(or a comment like `// Check for the same method in batches too`).

Implementing the logic for `call` and `batch` in these exmaples might lead to some ideas or whatever to make it easier to reuse logic across the two places, because that's a new challenge for people with this API that we might be able to expose some helper method or whatever to help with :)